### DISCUSSION TITLE
Playable

### DISCUSSION THREAD

**Discussion by kerimmerve** [skyline-emu/skyline#894] [General]
games are called "playable" is it only 2D games?
when are the 3D games due?

Google traduction

**Comment by jmtuch** [comment]
It will still take time to emulate 3D games. And there are only a few 2D ones that are currently playable.

**Comment by sauradipdebnath** [comment]
Unlikely check docs.google.com/spreadsheets/u/0/d/1oFND6f0vcQKsb8dMiVuhrwY-_Tkzf2RyEbjBM64b_eM/htmlview#gid=0

**Comment by Milborne** [comment]
Celeste 15fps - Motorola G30 - Snapdragon 665 - 4gb Ram

**Reply by iniMerchant64** [reply]
We already have that one on the list
